### ISSUE TITLE
Third Party Users Are Seeing "HPDC Collaborative" in Messages that Should Have Their Company Names

### ISSUE THREAD

**Issue by JEGeyer** [HPDCollaborative/builder-ui-beta-reporting#62]
**Describe the bug**
Approved Verifiers and Approved Preparers are not getting their company names added to Verifier/Preparer elements in the Builder.
**To Reproduce**
1. Go to the edit substance page as an approved verifier/preparer (I've asked Tara for a specific record where she's seeing this.) Screen shot below shows how it displays in an HPDC record.
2. Scroll down to check box at the bottom of the screen

1. Go to Contact information & Publication
2. Scroll to the bottom of the page page where the Preparer/Verifier check boxes are displayed.
**Expected behavior**
In each case, it should show the name of the preparer/verifier that is working on the HPD.

**Screenshots**
![third party screenshot 1](https://user-images.githubusercontent.com/34175254/46382899-45815a80-c662-11e8-962b-d5757b9e0936.png)

![third party screenshot 2](https://user-images.githubusercontent.com/34175254/46383069-4c5c9d00-c663-11e8-97da-f380b7846c9c.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome

**Smartphone (please complete the following information):**

**Additional context**

**Comment by JEGeyer** [comment]
It looks like HPDC is showing in all the records, no matter who the preparer/verifier is.

**Comment by 19peaches** [comment]
yeah that's totally my fault, this is the case on the references as well, it's because I copied and pasted the form. I have no idea how to fix this, I'll have to dig into the old code and see how they did it and see if I can find a work around for it.

**Comment by 19peaches** [comment]
also I will need to know how to set up my account as a preparer/verifier/consultant or whatever is the case here.

**Comment by Millali** [comment]
Hi Vince, 

This is how it works: 

**Account access levels  may be assigned in two places:** 

1. When Account Admin adds a team members to the account
2. When someone joins the account (without being invided)  and the admin approves them into the account and subsequently assinges an access level.  

To do 1: 
a. Go to Dashboard and click on "Add team member"
https://builder.hpd-collaborative.org/account/invite
b. Add email address and select the access level option. 
c. As you may notice in the options, there is a request for marking the checkbox for consultants. To be honest I don't understand why the consultant is checkbox and not another radio button. It is very confusing to users so if you can make it a radio button will be ideal!!!!. I believe that the only difference between a consultant and a View and Edit user is that the Consultant can add new products and the view and edit user can't. 

To do number 2: 
In this case a users has singed up for the Builder and has selected an existing company in the Builder as his primary company (with an already assigned admin). In this case the new user gets a message that tells him he needs to wait for the admin to approve him to join the account . The admin gets a notification of the new user and assigns the level of access for this user. Actually as an admin he can change the levels of admins of the other account users any time. To do this:

a. Go to Dashboard 
b. In the Users box, click on the name of other team members and select the right option. 
https://builder.hpd-collaborative.org/account/change/310
Weirdly there isn't an option for consultants here :s. Ussually consultant are invited but I think it should be an option here too. 

**Verifiers and Reviewers access is only grande by Super Admin.** To edit verifiers right follow the steps below: 

1. In the admin panel go to Companies
2. Select the company you want to modify, let say Healthy Building Network 
https://builder.hpd-collaborative.org/companies/1
3. Go to 3rd party privileges  and click Add.
4. Select the option you want to assign. 


Hope all of this helps!!! Let me know if you have questions! 

Best,

Milla

**Comment by 19peaches** [comment]
Ok well I don't have time to weed through all that, perhaps you can teach me at some point, but for now, I have a non-super admin, company admin account on my local dev, how turn that into whatever it needs to be to see the errors?

I make a view only account, then how do I add that user to another account so I can reproduce this situation?

**Comment by 19peaches** [comment]
Ok I read through all that and none of it applies to an existing account that I can see. I've been all through this app as a super admin and I don't see any way to make a user a consultant.
